kit/exercise/multiple_choice.go

```diff
@@ -28,14 +28,19 @@ func ParseMarkdownAnswers(answerFile string) (map[int]string, error) {
 	currentQ := -1
 	// map: question# -> answer label
 	answerMap := make(map[int]string)
+	multipleAnswers := make(map[int]bool)
 	for _, line := range strings.Split(string(md), "\n") {
 		if qNumRegExp.MatchString(line) {
 			qNum := qNumRegExp.ReplaceAllString(line, "$1")
 			// ignore error since regular expression ensure it is already a number
 			currentQ, _ = strconv.Atoi(qNum)
 		}
-		_, found := answerMap[currentQ]
-		if !found && currentQ != -1 && selectionRegExp.MatchString(line) {
+		if !multipleAnswers[currentQ] && currentQ != -1 && selectionRegExp.MatchString(line) {
+			if _, found := answerMap[currentQ]; found {
```

Comment:
Add comment here: `// if found is true, multiple answers are given for the same question`

Maybe also point out in a second doc line to the `ParseMarkdownAnswers` function:

```text
Only a single answer per question is allowed.
```